resources/lang/ko.js

```diff
@@ -84,21 +84,57 @@ class CactbotLanguageKo extends CactbotLanguage {
       Pull: '풀링',
     });
 
+    this.kPetNames = Object.freeze([
+      // Pulled from https://github.com/Ra-Workspace/ffxiv-datamining-ko/blob/master/csv/Pet.csv
+      '카벙클 에메랄드',
+      '카벙클 토파즈',
+      '이프리트 에기',
+      '타이탄 에기',
+      '가루다 에기',
+      '요정 에오스',
+      '요정 셀레네',
+      '자동포탑 룩',
+      '자동포탑 비숍',
+      '데미바하무트',
+      '데미피닉스',
+      '세라핌',
+      '카벙클 문스톤',
+      '영웅의 환영',
+      '자동인형 퀸',
+      '분신',
+    ]);
+
     // TODO: Countdowns
     this.countdownStartRegex = function() {
-      return Regexes.parse(/전투 시작 (\y{Float})초 전!/);
+      return Regexes.gameLog({
+        capture: true,
+        line: '전투 시작 (?<time>\\y{Float})초 전!',
+      });
     };
     this.countdownEngageRegex = function() {
-      return /:전투 시작!/;
+      return Regexes.gameLog({
+        capture: true,
+        line: '전투 시작!',
+      });
     };
     this.countdownCancelRegex = function() {
-      return / 님이 초읽기를 취소했습니다\./;
+      return Regexes.gameLog({
+        capture: true,
+        // FIXME Missing name capture
```

Comment:
@Jaehyuk-Lee Sorry for the ping, but do you have a full example line for the countdown being cancelled in Korean, so the name of who cancelled it can be captured?